Tasks/cURLUploader/curluploader.ts

```diff
@@ -108,8 +110,22 @@ async function run() {
             curlRunner.arg(userPassCombo);
         }
 
+        let output:string = '';
+        curlRunner.on('stdout', (buffer: Buffer) => {
+            process.stdout.write(buffer);
+            output = output.concat(buffer ? buffer.toString() : '');
+        });
+
         var code: number = await curlRunner.exec();
         tl.setResult(tl.TaskResult.Succeeded, tl.loc('CurlReturnCode', code));
+
+        let outputMatch:RegExpMatchArray = output.match(/[\n\r]100\s/g);
```

Comment:
Not a dumb question.  This is an 'either-or' match, so, "this *should* work for Win, Linux, and macOS," I say with all the confidence of someone who has only tested it on Windows.